CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # Changelog
 
+## 23.0.0 - 2023-08-16
+
+### StripeClient
+
+`StripeClient` and the new service-based pattern is introduced. `StripeClient` has multiple benefits over existing resource-based patterns:
```

Comment:
Wanted to call out that for php services we put a [special wiki entry](https://github.com/stripe/stripe-php/wiki/Migration-to-StripeClient-and-services-in-7.33.0) to describe the migration. Should we do that here too?

Reply:
I don't have a strong opinion. My plan was to copy changelog entry to the major release wiki page.